src/io/flutter/jxbrowser/EmbeddedBrowser.java

```diff
@@ -144,6 +147,10 @@ public void updatePanelToWidget(String widgetId) {
         LOG.error(ex);
         return;
       }
+      if (url == null) {
```

Comment:
no need to log with an error if this is something intentional we are causing if the app got shut.